Supertext.Base.Hosting.Specs/Queuing/QueuedHostedServiceTest.cs

```diff
@@ -77,7 +77,7 @@ public async Task QueueBackgroundWorkItem_BackgroundTaskIsDequeued_BackgroundWor
         public async Task QueueBackgroundWorkItem_WorkitemThrowsException_EmailIsSent()
         {
             EmailInfo sentEmailInfo = null!;
-            A.CallTo(() => _mailService!.SendAsync(A<EmailInfo>._)).Invokes(invocation => sentEmailInfo = invocation.GetArgument<EmailInfo>(0)!);
+            A.CallTo(() => _mailService!.SendAsync(A<EmailInfo>._, CancellationToken.None)).Invokes(invocation => sentEmailInfo = invocation.GetArgument<EmailInfo>(0)!);
```

Comment:
Wouldn't `A<CancellationToken>._` be safer? The method under test accepts a `CancellationToken`.

Reply:
Is more generic yes. But I wanted to leave this, to prove that it is no major change.